packages/api-cli/src/index.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
```

Comment:
Ideally this file should not be duplicated between the 2 packages.

Alternative, could this work? https://babeljs.io/docs/en/babel-node. So that `yarn run:api` runs `babel-node packages/api-cli/api.ts`, and potentially this file is not needed.

Reply:
There is method to the madness - would like to publish this and allow install with `npm -g` in which case we need an executable entry point per package.

Reply:
Ah, I see.